.github/workflows/ci.yaml

```
@@ -1,13 +1,19 @@
 name: CI
-on: push
+on:
+  - push
+  - pull_request
 jobs:
   test:
-    runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
+        os:
+          - ubuntu-20.04
+          - windows-2019
         node:
           - '18'
           - '16'
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
```

This needs to be updated also:

```suggestion
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
```

@tpluscode Can you do it also for the `lint` job?
And put version `node-version: 20` instead of `node-version: 18`?